ginkgo/utils/migration/applicationInventory/application.go

```diff
@@ -0,0 +1,20 @@
+package applicationinventory
+
+import (
+	"fmt"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func CreateMultipleApplications(numberOfApplications int) []api.Application {
+	var appSlice []api.Application
+	for i := 0; i < numberOfApplications; i++ {
+		// Create new application
+		application := api.Application{Name: fmt.Sprintf("test-app%d", i+1)}
```

Comment:
I'd suggest consider using https://github.com/konveyor/go-konveyor-tests/blob/main/hack/uniq/name.go#L26 like https://github.com/konveyor/go-konveyor-tests/blob/main/analysis/analysis_test.go#L30-L31

For some cases, it might be helpful use also predefined sample apps from https://github.com/konveyor/tackle2-hub/blob/main/test/api/application/samples.go#L10

Reply:
Done in e709088